messaging-api.yml

```
@@ -3566,6 +3567,92 @@ components:
           type: string
         emojiId:
           type: string
+    TextMessageV2:
+      externalDocs:
+        url: "https://developers.line.biz/en/reference/messaging-api/#text-message-v2"
+      type: object
+      required:
+        - textV2
```

```suggestion
        - text
```

Shouldn't the required property name be `text`?

---

Thank you for the review!